### ISSUE TITLE
react-template-helper, Meteor 1.3: Arguments to path.resolve must be strings

### ISSUE THREAD

**Issue by jiku** [meteor/react-packages#178]
Adding react-template-helper 0.2.6 to Meteor 1.3rc1 here yields

```shell
=> Errors prevented startup:

   While processing files with cosmos:browserify (for target web.browser):
   packages/react-runtime-dev/react.browserify.js: Arguments to path.resolve must be strings

   Browserify options:
   >  undefined

   While processing files with cosmos:browserify (for target os.osx.x86_64):
   packages/react-runtime-dev/react.browserify.js: Arguments to path.resolve must be strings

   Browserify options:
   >  undefined
```

Repro steps

```shell
mkdir repro && cd $_
meteor create .
meteor update --release=1.3-rc.1
npm init .
meteor add react-template-helper
meteor add blaze # optional
npm i react --save-dev # optional
meteor
```

Should add that I don't even know if react-template-helper going to be supported for 1.3+?
Opening here since I'm not sure if it's a package or Meteor issue.

**Comment by tmeasday** [comment]
It's weird, somehow I published `0.2.5` of this again as `0.2.6`, thus clobbering my `0.2.6-beta.16` version (which is the version that runs with 1.3).

Just republished as `0.2.7-rc.2`. Try `meteor add react-template-helper@0.2.7-rc.2`

**Comment by jiku** [comment]
Seems it's working with RC2. :) Thanks!
